RadixWallet/Features/TransactionReviewFeature/TransactionReview.swift

```diff
@@ -596,6 +627,8 @@ extension TransactionReview {
 			let sections = try await sections(for: executionSummary, networkID: networkID)
 			await send(.internal(.updateSections(sections)))
 		} catch: { error, send in
+			print("•  Failed to extract transaction: \(error)")
```

Comment:
loggerGloba.error ?

Reply:
We already have that, forgot to remove this debug code